config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md) for this project, " `Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route)` " please kindly create a route for your greetings resource.